static/js/containers/DashboardPage.js

```diff
@@ -475,6 +477,15 @@ class DashboardPage extends React.Component {
     )(this.props);
   };
 
+  showToPearsonSiteDialog = (open: boolean) => {
+    const { dispatch } = this.props;
+    if (open) {
+      dispatch(showDialog("showPearsonTosDialog"));
```

Comment:
it would be good to declare the string we pass to `showDialog` as a constant, maybe just at the top of this file. something like `PEARSON_TOS_DIALOG = 'PEARSON_TOS_DIALOG'`.